src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1297,39 +1316,40 @@ $$
 \epsilon_n \sim \textsf{normal}(0,\sigma).
 $$
 
-If either vector $x$ or $y$ has very large or very small values or if the
+If vector $x$ has very large or very small values or if the
 sample mean of the values is far away from 0 (on the scale of the values),
-then it can be more efficient to standardize the outputs $y_n$ and
-predictors $x_n$.  The data are first centered by subtracting the
+then it can be more efficient to standardize the
+predictors $x_n$, so that it has a mean of zero and standard deviation of 1.
+The data are first centered by subtracting the
 sample mean, and then scaled by dividing by the sample deviation.
 Thus a data point $u$ is standardized with respect to
-a vector $y$  by the function $\textsf{z}_y$, defined by
+a vector $x$  by the function $\textsf{z}_x$, defined by
 $$
-\textsf{z}_y(u) = \frac{u - \bar{y}}{\texttt{sd}(y)}
+\textsf{z}_x(u) = \frac{u - \bar{x}}{\texttt{sd}(x)}
 $$
-where the sample mean of $y$ is
+where the sample mean of $x$ is
 $$
-\bar{y}
-= \frac{1}{N} \sum_{n=1}^N y_n,
+\bar{x}
+= \frac{1}{N} \sum_{n=1}^N x_n,
 $$
-and the sample standard deviation of $y$ is
+and the sample standard deviation of $x$ is
 $$
-\texttt{sd}(y)
+\texttt{sd}(x)
 = \left(
-\frac{1}{N} \sum_{n=1}^N (y_n - \bar{y})^2
+\frac{1}{N} \sum_{n=1}^N (x_n - \bar{x})^2
```

Comment:
I don't think you need to define mean and sd in this context, though it's OK to leave them.

Reply:
revised.